features/karaf/openhab-core/src/main/feature/feature.xml

```diff
@@ -609,7 +609,7 @@
   </feature>
 
   <feature name="openhab-transport-serial" description="Serial Transport" version="${project.version}">
-    <bundle>mvn:org.openhab/nrjavaserial/${nrjavaserial.version}</bundle>
+    <feature>openhab.tp-serial-rxtx-noliblockdev</feature>
```

Comment:
Don't inject external dependencies but use a "target platform feature".
We do not use `dependency="true"` here because we want to force the usage of that specific serial implementation.